File: more_exercises/01_first_steps_in_coding/07_house_painting.py
x = float(input())
y = float(input())
h = float(input())
green_paint_liter = 3.4
red_paint_liter = 4.3
area_fwd_aft = (x * x - 1.2 * 2) + x * x
area_side = 2 * (x * y - 1.5 * 1.5)
area_roof_fwd_aft = x * h  #Може да се напише и така: 2 * x * h / 2 или 2 * ((x * h) / 2)
area_roof_side = 2 * (x * y)
total_paint_walls = (area_fwd_aft + area_side) / 3.4
total_paint_roof = (area_roof_fwd_aft + area_roof_side) / 4.3
print(f"{total_paint_walls:.2f}")
print(f"{total_paint_roof:.2f}")

File: 05_JavaScriptBasics/03_MoreExercises/01_FirstStepsInCoding/07_housePainting.js
function housePainting(input){
    let x = Number(input[0])
    let y = Number(input[1])
    let h = Number(input[2])

    let greenPaintLiter = 3.4
    let redPaintLiter = 4.3

    let areaFwdAndAft = (x * x - 1.2 * 2) + x * x
    let areaSides = 2 * (x * y - 1.5 * 1.5)
    let areaRoofFwdAft =  x * h  //Може да се напише и така: 2 * x * h / 2 или 2 * ((x * h) / 2)
    let areaRoofSides = 2 * (x * y)
    let totalPaintWalls = (areaFwdAndAft + areaSides) / greenPaintLiter
    let totalPaintRoof = (areaRoofFwdAft + areaRoofSides) / redPaintLiter

    console.log(totalPaintWalls.toFixed(2))
    console.log(totalPaintRoof.toFixed(2))
}

housePainting(["6", "10", "5.2"])
housePainting(["10.25", "15.45", "8.88"])
